docs/internals/layout_in_storage.rst

```diff
@@ -81,15 +81,15 @@ two mappings next to each other, their content ends up at different storage loca
 Array data is located starting at ``keccak256(p)`` and it is laid out in the same way as
 statically-sized array data would: One element after the other, potentially sharing
 storage slots if the elements are not longer than 16 bytes. Dynamic arrays of dynamic arrays apply this
-rule recursively. The location of element ``x[i][j]``, where the type of ``x`` is ``uint32[][]``, is
+rule recursively. The location of element ``x[i][j]``, where the type of ``x`` is ``uint24[][]``, is
 computed as follows (again, assuming ``x`` itself is stored at slot ``p``):
-The slot is ``keccak256(keccak256(p) + i) + floor(j / (256 / 32))`` and
-the element can be obtained from the slot data ``v`` using ``(v >> ((j % (256 / 32)) * 32)) & type(uint32).max``.
+The slot is ``keccak256(keccak256(p) + i) + floor(j / (256 / 24))`` and
+the element can be obtained from the slot data ``v`` using ``(v >> ((j % (256 / 24)) * 24)) & type(uint24).max``.
```

Comment:
For `uint24` the formula needs to be updated to account for the gaps.

Reply:
you mean add `floor` around the divisions?

Reply:
Yeah. I see that just the inner floor was missing. Looks fine now.